File: SeatMe-Backend/src/main/java/edu/icet/ecom/dto/Reservation.java
package edu.icet.ecom.dto;

import lombok.*;

import java.time.LocalDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Reservation {
    private Long id;
    private Long customerId;
    private Long tableId;
    private LocalDateTime reservationTime;
    private Integer numberOfGuests;
}

File: SeatMe-Frontend/src/app/model/Reservation.ts
export class Reservation {
    id: number;
    customerId: number;
    tableId: number;
    reservationTime: string;
    numberOfGuests: number;
  
    constructor(
      id: number,
      customerId: number,
      tableId: number,
      reservationTime: string,
      numberOfGuests: number
    ) {
      this.id = id;
      this.customerId = customerId;
      this.tableId = tableId;
      this.reservationTime = reservationTime;
      this.numberOfGuests = numberOfGuests;
    }
  }
  